omim2obo/main.py

```diff
@@ -164,63 +164,69 @@ def omim2obo(use_cache: bool = False):
+                LOG.warning(gene_label_err)  # todo: decide the best way to handle these situations
+            graph.add((omim_uri, RDFS.label, Literal(pref_symbols[0])))
         else:
-            graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(label))))
-
-        # Reify on abbreviations. See: https://github.com/monarch-initiative/omim/issues/2
-        exact_labels = [s.strip() for s in label.split(';')]
-        if len(exact_labels) > 1:
-            abbr = exact_labels.pop()
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbr)))
-            # Acronym: uppercase, no numbers, no whitespace, <10 chars
-            if abbr.isupper() and not any(char.isspace() for char in abbr) and len(abbr) < 10:
-                axiom = BNode()
-                graph.add((axiom, RDF.type, OWL.Axiom))
-                graph.add((axiom, OWL.annotatedSource, omim_uri))
-                graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
-                graph.add((axiom, OWL.annotatedTarget, Literal(abbr)))
-                graph.add((axiom, oboInOwl.hasExactSynonym, MONDONS.abbreviation))
-        for exact_label in exact_labels:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(exact_label, abbrev))))
-        for label in other_labels:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(label, abbrev))))
+            graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(pref_title))))
+
+        # todo: .clean() should accept all symbols/abbrevs (preferred, alt, included), else [] instead of str/None
+        #  - See: https://github.com/monarch-initiative/omim/issues/129
+        abbrev: Union[str, None] = None if not pref_symbols else pref_symbols[0]
+
+        # Add synonyms
+        graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(pref_title, abbrev))))
+        for alt_label in other_labels:
+            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(alt_label, abbrev))))
+        for abbreviation in pref_symbols:
+            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbreviation)))
+            # Reify on abbreviations. See: https://github.com/monarch-initiative/omim/issues/2
+            axiom = BNode()
+            graph.add((axiom, RDF.type, OWL.Axiom))
+            graph.add((axiom, OWL.annotatedSource, omim_uri))
+            graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
+            graph.add((axiom, OWL.annotatedTarget, Literal(abbreviation)))
+            graph.add((axiom, OBOINOWL.hasSynonymType, MONDONS.abbreviation))
+
+        # Add 'included' entry properties
         for included_label in cleaned_inc_labels:
             graph.add((omim_uri, URIRef(INCLUDED_URI), Literal(label_cleaner.clean(included_label, abbrev))))
+        if label_endswith_included_alt or label_endswith_included_inc:
```

Comment:
Let's check that moving this line here does not change which entries have the `included_detected_comment` comment.

Reply:
As with [this thread](https://github.com/monarch-initiative/omim/pull/130#discussion_r1748276110), I'm pretty sure this change was already there when we looked at the `.ofn` diffs, etc, on the call yesterday, so all should be well. But I did [run and upload another output](https://github.com/monarch-initiative/omim/pull/130#discussion_r1748972938), so feel free to check!

Reply:
FYI I think these additional comments, along w/ some re-ordering, make the code much more readable!